### ISSUE TITLE
DNSMOS for 48k

### ISSUE THREAD

**Issue by SiyuanWei** [microsoft/DNS-Challenge#77]
Hello, I saw that the dnsmos.py only supports 16k audio but the 4th DNS challenge dataset is full-band.
will there be a DNS-MOS API that supports 48k or should I resample the testclips to 16k and use the current API?

**Comment by motus** [comment]
@SiyuanWei I don't think we have a 48kHz DNSMOS model. Please resample to 16kHz. @vishakg please tell me if I am wrong

**Comment by motus** [comment]
P.S. @SiyuanWei it looks like we resample to 16kHz in the DNSMOS service anyway, so you don't really *have* to resample, but it would be faster and less load for us if you do.

**Comment by yanglei-im** [comment]
I‘m confused about this. Does it mean the evaluation tools only support 16kHz, but the challenge is for 48kHz? If so, does 48kHz NS for the 2 tracks necessary?

**Comment by motus** [comment]
@yanglei-im yes, the challenge is for 48kHz - that is, your model should take and produce signal sampled at 48kHz. Our current DNSMOS model uses 16kHz-sampled input, but this can change in the future. Also, besides the DNSMOS model we also use the speech recognition word error rate as a metric to evaluate models. I'll keep this issue open for a while - maybe @vishakg will add more details later

**Comment by vishakg** [comment]
We don’t have a 48khz DNSMOS API yet. Current API only supports 16khz input. Our tests indicate that the resampled evaluation of 48khz clips still hold good correlation with human raters.
That being said we are working on a 48khz model that we will make available soon. Unfortunately we don’t have an ETA that I can share for that at the moment.
